app/controllers/reviews_controller.rb

```diff
@@ -112,16 +94,36 @@ def parse_before_create(review_params)
     review_params
   end
 
-  def review_params
-    params.require(:review).permit(
-      :title, :comment,
-      :price, :star_rating,
-      :how_much, :how_much_custom,
-      :how_long, :how_long_custom,
-      :type_of_contact,
-      :space_id,
-      :organization
-      # facility_reviews_attributes: %i[facility_id experience id]
-    )
+  def review_params # rubocop:disable Metrics/MethodLength
+    case params[:review][:type_of_contact]
+    when "been_there"
+      params.require(:review).permit(
+        :title,
+        :comment,
+        :price, :star_rating,
+        :how_much, :how_much_custom,
+        :how_long, :how_long_custom,
+        :type_of_contact,
+        :space_id,
+        :organization
+      )
+    when "not_allowed_to_use"
+      params.require(:review).permit(
+        :title,
+        :comment,
+        :type_of_contact,
+        :space_id,
+        :organization
+      )
+    when "only_contacted"
+      params.require(:review).permit(
+        :comment,
+        :type_of_contact,
+        :space_id,
+        :organization
+      )
+    else
+      raise "No valid type of contact given. Check reviews controller."
+    end
```

Comment:
These could probably be split up into functions like params_been_there etc... That way, Rubocop will be satisfied  :slightly_smiling_face:

Reply:
You're right, but I don't think it's worth doing at this moment as this is probably as clear to read as that would be.